src/transformer/events/mod_quiz/question_answered/truefalse.php

```diff
@@ -53,7 +53,8 @@ function truefalse(array $config, \stdClass $event, \stdClass $questionattempt,
             'completion' => $questionattempt->responsesummary !== null,
             'success' => $questionattempt->rightanswer === $questionattempt->responsesummary,
             'extensions' => [
-                'http://learninglocker.net/xapi/cmi/true-false/response' => $questionattempt->responsesummary === 'True',
+                'http://learninglocker.net/xapi/cmi/true-false/response' => $questionattempt->responsesummary ===
+                    'True',
```

Comment:
I think this accidentally got moved onto a new line. Shouldn't affect functionality, I'll just move it back when I see it.

Reply:
Ah makes sense, it's all good 👍